Documentation/devel/user-namespaces.md

```diff
@@ -7,28 +7,28 @@ It can provide a better isolation and security: the privileged user `root` in th
 
 ## Implementation status
 
-rkt has an experimental implementation based on systemd-nspawn.
+rkt's implementation is based on systemd-nspawn.
 A pod can transparently use user IDs in the range 0-65535 and this range is mapped on the host to a high range chosen randomly.
 
 Before the pod is started, the ACIs are rendered to the filesystem and the owners of the files are set with `chown` in that high range.
 
-## Future work
+## Current limitations
 
-### Choosing the UID range
+### UID range allocation
 
 When starting several pods with user namespaces, they will each get a random UID range.
 In order to avoid collisions, it is planned to implement a locking
 mechanism so that two pods will always have a different UID range.
 
-### Working with overlayfs
+### Incompatible with overlayfs
 
 The initial implementation works only with `--no-overlay`.
 Ideally, preparing a pod should not have to iterate over all files to call `chown`.
 
 It is planned to add kernel support for a mount option to shift the user IDs in the correct range (https://github.com/coreos/rkt/issues/1057).
 It would make it work with overlayfs.
 
-### Volumes
+### Inconvenient UID shift on volumes
 
 When mounting a volume from the host into the pod, the ownership of the files is not shifted, so it makes volumes difficult if not impossible to use with user namespaces.
```

Comment:
Not for volumes passed with `--volume` & `--mount`. We only do the recursive chown on the files from the ACI.